ignite/services/chain/generate.go

```diff
@@ -160,7 +164,13 @@ func (c *Chain) Generate(
 			vuexPath = defaultVuexPath
 		}
 
-		vuexPath = filepath.Join(c.app.Path, vuexPath, "generated")
+		if filepath.IsAbs(vuexPath) {
+			// TODO: Should we always generate Vuex code inside a "generated" directory?
+			vuexPath = filepath.Join(vuexPath, "generated")
+		} else {
+			vuexPath = filepath.Join(c.app.Path, vuexPath, "generated")
+		}
```

Comment:
I feel like this could be refactored into a small util function since the logic is duplicated.

```go
func (c chain) generatedPath(string path) {
...
}
```